module/actor/vampire-actor-sheet.js

```diff
@@ -0,0 +1,193 @@
+/* global game, mergeObject */
+
+import { GhoulActorSheet } from './ghoul-actor-sheet.js'
```

Comment:
So a Vampire is mostly a ghoul with hunger, blood potency and better disciplines :)